packer.json

```diff
@@ -1,7 +1,9 @@
 {
   "variables": {
     "aws_access_key": "",
-    "aws_secret_key": ""
+    "aws_secret_key": "",
```

Comment:
I don't think this is necessary.  Packer should complain if this isn't defined in the environment variables.